workspaces/client/app/constant.ts

```diff
@@ -8,3 +8,5 @@ export const GRADE = [
 
 export const BIO_MAX_LENGTH = 255; // Bioは255文字まで許容
 export const BIO_MAX_NEWLINE = 10; // Bioは10行まで許容
+
+export const RESERVED_WORDS = ["maximum", "home", "calender", "members", "logout", "login", "meline", "merin", "idp"];
```

Comment:
```suggestion
export const RESERVED_WORDS = ["maximum", "home", "calendar", "member", "members", "logout", "login", "meline", "merin", "idp"];
```
- calender -> calendar
- member も追加で